diff --git a/node_modules/expo-live-activity/ios/Helpers.swift b/node_modules/expo-live-activity/ios/Helpers.swift
index b9fa67f..d1576fb 100644
--- a/node_modules/expo-live-activity/ios/Helpers.swift
+++ b/node_modules/expo-live-activity/ios/Helpers.swift
@@ -1,7 +1,7 @@
 func resolveImage(from string: String) async throws -> String {
   if let url = URL(string: string), url.scheme?.hasPrefix("http") == true,
      let container = FileManager.default.containerURL(
-       forSecurityApplicationGroupIdentifier: "group.expoLiveActivity.sharedData"
+       forSecurityApplicationGroupIdentifier: "group.com.aligntracker.app"
      )
   {
     let data = try await Data.download(from: url)
